docs/metrics.md

```diff
@@ -8,7 +8,8 @@ The `MetricsCollector` exposes the following metrics:
 
 - `pepr_errors`: A counter that increments when an error event occurs in the application.
 - `pepr_alerts`: A counter that increments when an alert event is triggered in the application.
-- `pepr_summary`: A summary that provides the observed durations of events in the application.
+- `pepr_Mutate`: A summary that provides the observed durations of mutation events in the application.
```

Comment:
does it matter that these are capitalized and the previous one wasn't?

Reply:
welllllll. I'm using Mutate and Validate strings internally, so we could lower case them, If this is killing you, we can do that.